tests/integration/cartography/intel/gcp/test_crm.py

```diff
@@ -6,20 +6,81 @@
 
 
 def test_load_gcp_projects(neo4j_session):
+    """
+    Tests that we correctly load a sample hierarchy chain of GCP organizations to folders to projects.
+    """
+    cartography.intel.gcp.crm.load_gcp_organizations(
+        neo4j_session,
+        tests.data.gcp.crm.GCP_ORGANIZATIONS,
+        TEST_UPDATE_TAG,
+    )
+    cartography.intel.gcp.crm.load_gcp_folders(
+        neo4j_session,
+        tests.data.gcp.crm.GCP_FOLDERS,
+        TEST_UPDATE_TAG,
+    )
     cartography.intel.gcp.crm.load_gcp_projects(
         neo4j_session,
         tests.data.gcp.crm.GCP_PROJECTS,
         TEST_UPDATE_TAG,
     )
 
+    # Ensure the sample project gets ingested correctly
     expected_nodes = {
-        ("sample-id-232323", "sample-number-121212"),
+        ("project-232323", "sample-number-121212"),
     }
-
     nodes = neo4j_session.run(
         """
         MATCH (d:GCPProject) return d.id, d.projectnumber
         """
     )
     actual_nodes = {(n['d.id'], n['d.projectnumber']) for n in nodes}
     assert actual_nodes == expected_nodes
+
+    # Expect (GCPProject{project-232323})<-[:RESOURCE]-(GCPFolder{1414})
+    #             <-[:RESOURCE]-(GCPOrganization{1337}) to be connected
+    query = """
+    MATCH (p:GCPProject{id:{ProjectId}})<-[:RESOURCE]-(f:GCPFolder)<-[:RESOURCE]-(o:GCPOrganization)
```

Comment:
I think we can test the parentless projects here, too, with `OPTIONAL MATCH`.

Reply:
Definitely can do that, but I'd rather keep that as a separate test so that it's easier to see if that case ever does fail.